pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -85,5 +92,70 @@ func status() registry.State {
 		return registry.State{Installed: true, Error: fmt.Errorf("%s failed:\n%s", strings.Join(cmd.Args, " "), out), Fix: "Run 'brew install hyperkit'", Doc: docURL}
 	}
 
+	// Split version from v0.YYYYMMDD-HH-xxxxxxx or 0.YYYYMMDD to YYYYMMDD
+	currentVersion := convertVersionToDate(string(out))
+	specificVersion := splitHyperKitVersion(GetHyperKitVersion())
+	// If hyperkit is not newer than specificVersion, suggest upgrade information
+	isNew, err := isNewerVersion(currentVersion, specificVersion)
+	if err != nil {
+		return registry.State{Installed: true, Error: fmt.Errorf("hyperkit version check failed:\n%s", err), Doc: docURL}
+	}
+	if !isNew {
+		return registry.State{Installed: true, Error: fmt.Errorf("the mininum recommended hyperkit version is %s. your hyperkit version is 0.%s. please consider upgrading your hyperkit", GetHyperKitVersion(), currentVersion), Fix: "Run 'brew upgrade hyperkit'", Doc: docURL}
```

Comment:
set `Healthy: true`, as we don't really know yet if the hyperkit driver is healthy or not even if the version is old.

This way the hyperkit driver can still be automatically selected, but the error displayed.

Reply:
> This way the hyperkit driver can still be automatically selected, but the error displayed.

Thanks. I added Healthy property.